osi_common_extension.proto

```diff
@@ -0,0 +1,305 @@
+syntax = "proto2";
+
+option optimize_for = SPEED;
+
+package osi3;
+
```

Comment:
Hmm.. I'm not sure at that point. In osi_common there are always units in the description (is this not the reasion why we standardize?)

Reply:
We do use SI units and yes it is a very important part of the standard.

Reply:
Then I guess the unit field of a traffic sign should only be filled if the unit is shown on the sign? Otherwise I don't see any reason why it exists then. And even then the documentation is not sufficient and I am sure it is misused.